lib/index.js

```diff
@@ -55,11 +55,10 @@ class Rooftop {
       if (transformFn === true) transformFn = transform
       else if (transformFn === false) transformFn = (x) => x
 
-      return this.client[name].get(options)
+      return W.resolve(this.client[name].get(options))
         .then((p) => W.map(p, transformFn))
-        .then((v) => m[name] = v)
-        .then(() => m)
-
+        .then((v) => { m[name] = v })
```

Comment:
done - tho can you explain why tap over then if we're not passing the value into a promise and yielding that value?